### PR TITLE
Ietf120 feedback

### PR DESCRIPTION
**Pull request by jasdips**
This PR address issues #10, #12, and #13.

### CODE REVIEW

File: draft-jasdips-regext-rdap-rpki.md [hunk 1]
```diff
@@ -74,72 +75,77 @@ Furthermore, correlating registered RPKI data with registered IP networks and au
 access to the latter's contact information through RDAP entity objects, which should aid troubleshooting.
 
 In addition to troubleshooting, serving RPKI meta-data over RDAP offers a convenience to network operators
-through a simple lookup mechanism. As is demonstrated in the [@?RDAP-GUIDE](https://rdap.rcode3.com/misc/uses.html),
-constructing custom RDAP scripts is relatively easy and beneficial to network operators for the purposes of
-reporting. Systems such as [@?JDR](https://blog.nlnetlabs.nl/introducing-jdr/) have shown the utility of such
-an approach. 
+through a simple lookup mechanism. As is demonstrated in [@RDAP-GUIDE], constructing custom RDAP scripts is
+relatively easy and beneficial to network operators for the purposes of reporting. Systems such as [@JDR] have shown the
```
Comment (ties) [inline draft-jasdips-regext-rdap-rpki.md]: I do not think JDR consumed RDAP. It was a piece of Julia code that parsed the RPKI objects. Will add a suggestion.

File: draft-jasdips-regext-rdap-rpki.md [hunk 2]
```diff
@@ -74,72 +75,77 @@ Furthermore, correlating registered RPKI data with registered IP networks and au
...
+utility of such an approach.
```
Comment (ties) [inline draft-jasdips-regext-rdap-rpki.md]:
```suggestion
utility of systems that allow users to explore the RPKI hierarchy in a visual fashion, without interacting with the signed objects directly.
```

(Could also mention the cloudflare explorer)

File: draft-jasdips-regext-rdap-rpki.md [hunk 3]
```diff
@@ -74,72 +75,77 @@ Furthermore, correlating registered RPKI data with registered IP networks and au
...
-* handle -- a string representing the registry unique identifier of the RPKI object registration
-* name -- a string representing an identifier assigned to the RPKI object registration by the registration holder
-* notValidBefore -- a string that contains the time and date in Zulu (Z) format with UTC offset of 00:00 ([@!RFC3339]), 
-  representing the not-valid-before date of the end-entity certificate for the RPKI object ([@!RFC6487, section 4])
-* notValidAfter -- a string that contains the time and date in Zulu (Z) format with UTC offset of 00:00 ([@!RFC3339]), 
-  representing the not-valid-after date of the end-entity certificate for the RPKI object ([@!RFC6487, section 4])
-* autoRenewed -- a boolean indicating if the registered RPKI object is auto-renewed if true
-* publicationUri -- a URI string pointing to the location of the RPKI object within the RPKI repository; 
+* "handle" -- a string representing the registry unique identifier of an RPKI object registration
+* "name" -- a string representing the identifier assigned to an RPKI object registration by the registration holder
```
Comment (ties) [inline draft-jasdips-regext-rdap-rpki.md:103]: `handle` and `name` will not be available in all systems. The RIPE NCC CA does not expose this kind of identifier to the user at the moment.

File: draft-jasdips-regext-rdap-rpki.md [hunk 4]
```diff
@@ -74,72 +75,77 @@ Furthermore, correlating registered RPKI data with registered IP networks and au
...
+* "publicationUri" -- a URI string pointing to the location of an RPKI object within an RPKI repository;
   the URI scheme is "rsync", per [@!RFC6487, section 4]
-* source -- a string representing the registry-unique identifier (handle) of the organization (entity) which is the authoritative source for the RPKI object
-* rpkiType -- a string literal representing the type of the RPKI repository, with the following possible values:
-    * "hosted" -- The Certificate Authority (CA) and the RPKI repository are operated by a registry for an organization with allocated resources
-    * "delegated" -- A repository and CA operated by an organization with resources allocated by a registry
-    * "hybrid" -- A repository operated by a registry for an organization with allocated resources in which the organization operates its own CA.
+* "entities" -- an array of entity objects ([@!RFC9083, section 5.1]), including the organization (entity) registered as
+  the authoritative source for an RPKI object
+* "rpkiType" -- a string literal representing various combinations of an RPKI repository and a Certificate Authority
```
Comment (ties) [inline draft-jasdips-regext-rdap-rpki.md]:
```suggestion
* "rpkiType" -- a string literal representing various combinations of an RPKI repository and a Certification Authority
```

**Review by ties — APPROVED** [review]

File: draft-jasdips-regext-rdap-rpki.md (hunk 4 above)
Comment (jasdips) [reply draft-jasdips-regext-rdap-rpki.md]: Thanks, will update.

File: draft-jasdips-regext-rdap-rpki.md (hunk 3 above)
Comment (jasdips) [reply draft-jasdips-regext-rdap-rpki.md:103]: Good point. For any RDAP object class, all members beside "objectClassName" are considered optional. We could label each member as REQUIRED/OPTIONAL but have shied away from that for now.

Comment (jasdips) [reply draft-jasdips-regext-rdap-rpki.md:103]: Logged this gap as issue #18.

File: draft-jasdips-regext-rdap-rpki.md (hunk 2 above)
Comment (jasdips) [reply draft-jasdips-regext-rdap-rpki.md]: Added Cloudflare RPKI Portal reference. Further, updated:
>Though not RDAP-based, systems such as [@JDR] and [@CLOUDFLARE] have shown the utility of an approach that allows users to explore the RPKI hierarchy in a visual fashion, without interacting with the signed objects directly.

File: draft-jasdips-regext-rdap-rpki.md (hunk 1 above)
Comment (anewton1998) [reply draft-jasdips-regext-rdap-rpki.md]: It didn't do RDAP, but it did allow getting the information via the web. The point here is that sometimes it is beneficial to get the information via another method.

**Pull request merged**